src/Components/codeMirror/extensions.ts

```diff
@@ -115,12 +116,14 @@ const LANG_EXTENSIONS: Record<string, () => Extension> = {
   javascript: javascript,
   html: html,
   css: css,
+  sql: sql,
   r: () => StreamLanguage.define(r),
 };
 
 export function getLanguageExtension(filetype: string | null): Extension {
   if (filetype === null) return [];
   if (!(filetype in LANG_EXTENSIONS)) return [];
+  console.log(`Loading language extension for ${filetype}`);
```

Comment:
Is there a reason to log this to the console or is this left over from debugging?

Reply:
oops, left over
```suggestion
```